app/templates/views/dashboard/review-email-list.html

```diff
@@ -32,7 +32,7 @@ <h1 class="heading-large py-4">{{ _('Review email addresses') }}</h1>
                 {% if bounce_rate["bounce_percentage_display"] < 0.1 %}
                   {{ _('Less than') }} 0.1%
                 {% else %}
-                  {{ bounce_rate["bounce_percentage_display"]|round(2) }}%
+                  {{ bounce_rate["bounce_percentage_display"]|round(1) }}%
```

Comment:
```suggestion
                  {{ bounce_rate["bounce_percentage_display"]|round(1)|string + "%" if session["userlang"] == 'en' else bounce_rate["bounce_percentage_display"]|round(1)|replace('.', ',')|string + " %"}}
```

Let's update to the correct french decimal format while we're at it